src/components/Cart/CartItems.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import { useCart } from "../../context/cart-context";
+import { useWishlist } from "../../context/wishlist-context";
+import "./Cart.css";
+export const CartItems = () => {
+  const { cartState, cartDispatch } = useCart();
+  const { wishlistState, wishlistDispatch } = useWishlist();
+
+  return (
+    <>
+      <div className="cart-product-container">
+        {cartState.map((product) => {
```

Comment:
- Can we use destuctuing to extract product object values, because you are doing product.```some property``` in this file multiple time.

- So before return statement we can do this ```const  {_id, img, title, author, quantity} = product;```